OOVisualization/src/expressions/VMetaCallExpression.h

```diff
@@ -60,6 +60,7 @@ class OOVISUALIZATION_API VMetaCallExpression : public Super<VExpression<VMetaCa
 		Visualization::Static* prefix_{};
 		Item* callee_{};
 		Visualization::VList* arguments_{};
+		Item* generated_{};
```

Comment:
This is just for debugging right? Please put a big TODO here to remove this once we're done. There should be an alternative mechanism for displaying the generated code. An on-demand mechanism, that's just a pop-up.